helper/resource/testing_new_import_state.go

```diff
@@ -93,11 +93,35 @@ func testStepNewImportState(ctx context.Context, t testing.T, helper *plugintest
 
 	logging.HelperResourceTrace(ctx, fmt.Sprintf("Using import identifier: %s", importId))
 
-	// Create working directory for import tests
+	// Prepare the test config dependent on the kind of import test being performed
 	if testStepConfig == nil {
 		logging.HelperResourceTrace(ctx, "Using prior TestStep Config for import")
 
-		testStepConfig = cfg
+		switch step.ImportStateKind {
+		case ImportBlockWithResourceIdentity:
+			t.Fatalf("TODO implement me")
+		case ImportBlockWithId:
+			cfgRaw += fmt.Sprintf(`
+			import {
+				to = %s
+				id = %q
+			}
+		`, step.ResourceName, importId)
+		default:
+			// Not an import block test so nothing to do here
+		}
+
+		confRequest := teststep.PrepareConfigurationRequest{
```

Comment:
I wonder how this interacts with the earlier `teststep.PrepareConfigurationRequest` from `step.Config` on line 26. Would this change the behavior of any existing `ImportCommandWithId` tests that define `step.Config`? I actually don't know 😃

Reply:
I can confirm that this doesn't change the behaviour of existing `ImportCommandWithId` tests that define a `Config` in-line.